example/transferAsset/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/CodeChain-io/codechain-sdk-go/core"
+	"github.com/CodeChain-io/codechain-sdk-go/core/transaction"
+	"github.com/CodeChain-io/codechain-sdk-go/key"
+	"github.com/CodeChain-io/codechain-sdk-go/primitives"
+	"github.com/CodeChain-io/codechain-sdk-go/rpc"
+)
+
+func main() {
+
+	rpcClient := rpc.NewRPC("http://localhost:8080")
+	networkID := "tc"
+
+	signerAccountID, err := primitives.PlatformAddressFromString("tccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6yamrrgd")
+	if err != nil {
+		fmt.Println(err)
+		return
```

Comment:
How about `panic(fmt.Println(err))` or `panic(err)`?

Reply:
I prefer panic(err). 
btw, Do you mean for every `fmt.Println(err)` ?

Reply:
![Screen Shot 2020-01-17 at 12 09 02 PM](https://user-images.githubusercontent.com/31873152/72581307-3dfec700-3922-11ea-82d9-350f0093bc08.png)
I think `panic(err)` shows enough message to the users.
The image is the result when I run the code with no codechain solo is running.

Reply:
Good!